bin/is-website-vulnerable.js

```diff
@@ -4,10 +4,17 @@
 
 const debug = require('debug')('is-website-vulnerable')
 const { Audit, RenderConsole } = require('../index')
+const url = require('url')
 
-const url = process.argv[2]
-debug(`received url argument: ${url}`)
-if (!url) {
+let urlToScan = process.argv[2]
+debug(`received url argument: ${urlToScan}`)
+
+// eslint-disable-next-line node/no-deprecated-api
+if (url.parse(urlToScan).protocol === null) {
+  urlToScan = 'http://' + urlToScan
```

Comment:
Should we default to trying over `https`? Perhaps if the tool is used to scan a URL which is served over `http` we can include a generic `Get yourself an SSL` message in the output.

Reply:
I just tested and If we add the `https` protocol by default, for the `non-https` website is not throwing any error (it should because the actual website with no http doesn't exist) and instead of that is scanning the website returning the default message of the scanner about `no vulnerabilities found` so we don't know when the non-https exists or not.

And is we use the `http` then we could succeed all the time.

Regarding the message `Get yourself an SSL` , I agree!

Reply:
I agree about defaulting to http and letting the remote webpage handling redirects as necessary to avoid failing on non-https endpoints.